standards.cfg

```diff
@@ -1,2 +1,4 @@
 # Core library, ensure code coverage is 100%
 PHPUNIT_COVERAGE_MINIMUM_LEVEL=100
+
+PHPSTAN_REPORTING_LEVEL=8
```

Comment:
I feel like there should be a comment here but I don't know why

Reply:
```
// A comment here
```
hah